appinfo/info.xml

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0"?>
 <info>
 	<id>twofactor_totp</id>
-	<name>Two Factor TOTP Provider</name>
-	<description>A Two-Factor-Auth Provider for TOTP (e.g. Google Authenticator)</description>
+	<name>2-Factor Authentication</name>
+	<summary>Increase access security with 2FA/TOTP</summary>
+	<description>2-Factor Authentication adds another layer of access security to ownCloud. The authentication method confirms a user’s identity utilizing a combination of two independent components. Instead of simply requiring a username/password combination, an additional access token (time-based one-time password/TOTP), which can be obtained via mobile phones for instance, is needed to gain access to ownCloud accounts. This not only improves access security, but also provides administrators with an option for disabling individual tokens. Time-based one-time passwords (TOTP) can be obtained through services like Google Authenticator or an Open-Source implementation of the TOTP standard like LinOTP.
+This extension provides the most simple way to add a second factor to user logins. More sophisticated 2-Factor Authentication mechanisms with additional authentication devices is available via the [PrivacyIDEA ownCloud App](https://marketplace.owncloud.com/apps/twofactor_privacyidea) or by integrating [other Identity Providers](https://marketplace.owncloud.com/apps/user_shibboleth).</description>
```

Comment:
Not sure if it is valid to link to other products?

Reply:
PrivacyIDEA is a partner app and user_shibboleth is our own. Both can be enhancements of simple TOTP. I don't see an issue :)

Reply:
I still don't really see the point as it was just a note on alternatives but respecting your community interest I removed the link.